pom.xml

```diff
@@ -112,6 +123,7 @@
                 <artifactId>formatter-maven-plugin</artifactId>
                 <version>2.16.0</version>
                 <configuration>
+                    <!--suppress UnresolvedMavenProperty -->
```

Comment:
What's this for?
I assume there's an unresolved maven prop, but which one and why would we include any such property?

Reply:
The one on the next line - jetbrains complains